docs/docs/en/guides/develop/build/registry.md

```diff
@@ -96,6 +104,10 @@ dependencies: [
 ]
 ```
 
+> [!NOTE]
```

Comment:
I'd consider removing the duplication of content across sections and having a section: "Package identifier convention". I'd then include examples of how some open-source package URLs map to the identifiers that we use.